docs/source/en/api/diffusion_pipeline.md

```diff
@@ -1,36 +0,0 @@
-<!--Copyright 2023 The HuggingFace Team. All rights reserved.
```

Comment:
Hmm where is the diffusion pipeline now shown instead? The `DiffusionPipeline` is a very important object no?

Reply:
The diffusion pipeline is shown [here](http://localhost:3000/api/pipelines/overview#diffusers.DiffusionPipeline) with the pipeline overview similar to how we show the base scheduler/model classes on their overview pages.